task_14/pageobjects/basepage.js

```diff
@@ -0,0 +1,17 @@
+var webdriver = require('selenium-webdriver')
+const {Select} = require('selenium-webdriver')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jkalandarov/epam_stage_2/pullRequest?prid=8887247)
```suggestion
const {Select} = require("selenium-webdriver")
```